lib/fetch/index.js

```diff
@@ -594,12 +580,9 @@ async function mainFetch (fetchParams, recursive = false) {
 
         // 4. If noCorsResponse is a filtered response or the CORB check with
         // request and noCorsResponse returns allowed, then return noCorsResponse.
-        if (noCorsResponse.status === 0 || CORBCheck(request, noCorsResponse) === 'allowed') {
-          return noCorsResponse
-        }
+        return noCorsResponse
 
         // 5. Return a new response whose status is noCorsResponse’s status.
-        return makeResponse({ status: noCorsResponse.status })
```

Comment:
I think this changes externally observable behavior

Reply:
`CORBCheck` would only return `allowed` so noCorsResponse was always being returned.